runbotics/runbotics-actions-windows/src/automation/office/excel.utils.ts

```diff
@@ -2,22 +2,33 @@ import { StartCellCoordinates } from "./excel.types";
 import ExcelErrorLogger from "./excelError.logger";
 
 export class ExcelUtils {
-    static switchWorksheet(getSession: () => any, setSession: (session: any) => void, setPrevWorksheet: (worksheet?: string) => void, worksheet?: string): void {
+    static switchWorksheet(getSession: () => any, setPrevWorksheet: (worksheet?: string) => void, worksheet?: string): void {
         const session = getSession();
         if (!session) return;
```

Comment:
session should not be possible to be null if we have a guard in switch/case clause